test/quantization/core/test_quantized_module.py

```diff
@@ -274,14 +274,26 @@ def _test_conv_api_impl(
             batch_size, in_channels_per_group, input_feature_map_size,
             out_channels_per_group, groups, kernel_size, X_scale, X_zero_point,
             W_scale, W_zero_point, use_bias, use_channelwise)
+
```

Comment:
there are a lot of if/else here in this function, I'm wondering if we can pass them as arguments,
e.g. can we pass 
1). inputs to the module as arguments ([[X_q, X2_q]] if (post_op == "add") else [[X_q]])
2). separate the fused_conv_module and the individual conv_module as two arguments, to avoid branches in L296-303
3). any other refactors to remove branches

Reply:
Thanks for the suggestions. I have removed most of the if/else branches in this function. Please help to take a look again @jerryzh168.